index.html

```diff
@@ -26,36 +26,191 @@ <h4>Rabaya Rabu</h4>
         <!-- headline section -->
         <section>
             <div class="headline">
-                <h1 class="h1">Hey I am Rabu</h1>
-
-                <h4 class="h4">I am a Software Developer</h4>
-
-                <p class="p">
+                <h1>Hey I am Rabu</h1>
+                <h4>I am a Software Developer</h4>
+                <p>
                     I can help you build a product , feature or website Look
                     through some of my work and experience! If you like what you
                     see and have a project you need coded,
                     don’t hestiate to contact me.
                 </p>
                 <!-- social media section -->
-                    <ul class="image-icons">
-                        <li><a href="https://github.com/rabayarabu"><img src="Images/GitHub.png"
-                                    alt="GitHub"></a></li>
-                        <li><a href="https://www.linkedin.com/in/rabaya-rabu-142721169/"><img src="Images/Linkedin.png"
-                                    alt="Linkedin"></a></li>
-                        <li><a href=""><img src="Images/Angelist.png"
-                                    alt="Angelist"></a></li>
-                        <li><a href="https://twitter.com/rabaya_rabu"><img src="Images/Twitter.png"
-                                    alt="Twitter"></a></li>
-                        <li><a href=""><img src="Images/Medium.png"
-                                    alt="Medium"></a></li>
+                <ul class="image-icons">
+                    <li><a href="https://github.com/rabayarabu"><img
+                                src="Images/GitHub.png"
+                                alt="GitHub"></a></li>
+                    <li><a
+                            href="https://www.linkedin.com/in/rabaya-rabu-142721169/"><img
+                                src="Images/Linkedin.png"
+                                alt="Linkedin"></a></li>
+                    <li><a href=""><img src="Images/Angelist.png"
+                                alt="Angelist"></a></li>
+                    <li><a href="https://twitter.com/rabaya_rabu"><img
+                                src="Images/Twitter.png"
+                                alt="Twitter"></a></li>
+                    <li><a href=""><img src="Images/Medium.png"
+                                alt="Medium"></a></li>
+                </ul>
+            </div>
+        </section>
+        <!-- works section -->
+        <section class="works">
+            <article class="grid-container">
+                <h1>My Recent Works</h1>
```

Comment:
Kindly update this element to use another `<hx>` element apart from `<h1>`🙏. This is for semantic reasons, as a good webpage is typically supposed to have one `<h1>` tag which represents the main title